mono/metadata/icall-def.h

```diff
@@ -439,6 +439,31 @@ ICALL(MATH_16, "Sqrt", ves_icall_System_Math_Sqrt)
 ICALL(MATH_17, "Tan", ves_icall_System_Math_Tan)
 ICALL(MATH_18, "Tanh", ves_icall_System_Math_Tanh)
 
+ICALL_TYPE(MATHF, "System.MathF", MATHF_1)
+ICALL(MATHF_1, "Acos", ves_icall_System_MathF_Acos)
```

Comment:
These can be HANDLES(..) I think.

Reply:
What do you mean?

Reply:
Many but not all lines icall-def.h have HANDLES() around them.
This is how icalls get converted to be coop-friendly.
1. passes/returns coop handles instead of raw pointers
2. passes an extra, initialized MonoError*
3. Check the error at the end.
Eventually every line should either have HANDLES or some comment or other macro explaining why not -- for example, maybe the only one, if no pointers and no errors and performance sensitive, that combination does not want HANDLES.

Reply:
@vargaz wouldn't HANDLE'izing those functions produce a net loss of perf for something they don't need? IE, none raise exceptions or take ref pointers.